### PR TITLE
Cleaning home page code.

### PR DESCRIPTION
**Pull request by walahamad**

### CODE REVIEW

File: src/com/generic/page/Cart.java [hunk 1]
```diff
@@ -125,6 +125,7 @@ public static String getTotalPrice() throws Exception {
 	}
 
 	//Done CBI
+	//TODO:: Duplicated from HomePage file.
```
Comment (ibatta) [inline src/com/generic/page/Cart.java]: for now, let us keep them as they are, just add on them TODO to return back to them later

File: src/com/generic/page/Cart.java [hunk 2]
```diff
@@ -135,6 +136,7 @@ public static boolean isListDisplayed(List<WebElement> elements) {
...
```
Comment (ibatta) [inline src/com/generic/page/Cart.java]: for now, let us keep them as they are, just add on them TODO to return back to them later

File: src/com/generic/page/HomePage.java [hunk 3]
```diff
@@ -122,19 +124,21 @@ public static void clickOnRandomAccountMenuItem() throws Exception {
...
-		((JavascriptExecutor) SelTestCase.getDriver()).executeScript("arguments[0].click()", element);
+		SelectorUtil.clickOnWebElement(element);
 		getCurrentFunctionName(false);
 	}
 
 	public static List<WebElement> getAccountMenuItems() throws Exception {
 		getCurrentFunctionName(true);
 		logs.debug("Get the account menu items.");
 		List<WebElement> menuItems = new ArrayList<WebElement>();
+		//TODO:: (use getAllElements in selectorUtils instead of this function.
```
Comment (ibatta) [inline src/com/generic/page/HomePage.java]: Please do this change

File: src/com/generic/page/HomePage.java [hunk 4]
```diff
@@ -122,19 +124,21 @@ public static void clickOnRandomAccountMenuItem() throws Exception {
...
 		menuItems = getElementsList(HomePageSelectors.accountMenuItems.get());
 		getCurrentFunctionName(false);
 		return menuItems;
 	}
 
+	//TODO:: No need for this function (use getAllElements in selectorUtils instead of this function.)
```
Comment (ibatta) [inline src/com/generic/page/HomePage.java]: Please do this change

File: src/com/generic/page/HomePage.java [hunk 5]
```diff
@@ -169,6 +173,8 @@ public static boolean validateGlobalFooterItemsDisplayed() throws Exception {
 		logs.debug("Validate if global footer  exist");
 		isDisplayed = SelectorUtil.isDisplayed(HomePageSelectors.globalFooter.get());
 		List<WebElement> footerItems = new ArrayList<WebElement>();
+
+		//TODO:: (use getAllElements in selectorUtils instead of this function.
```
Comment (ibatta) [inline src/com/generic/page/HomePage.java]: Please do this change

File: src/com/generic/page/HomePage.java [hunk 6]
```diff
@@ -505,6 +510,8 @@ public static boolean validateSubMenuLists(List<WebElement> menuFirstLevelElemen
 
 		logs.debug("Get the menu items first level.");
 		List<WebElement> menuFirstLevelElements = new ArrayList<WebElement>();
+
+		//TODO:: (use getAllElements in selectorUtils instead of this function.
```
Comment (ibatta) [inline src/com/generic/page/HomePage.java]: Please do this change

File: src/com/generic/page/HomePage.java [hunk 7]
```diff
@@ -578,6 +583,8 @@ public static boolean validateModalMenuSecondLevel() throws Exception {
 				currentPageMatchNavigated = false;
 				validateSubMenuNavigation = false;
 			} else {
+
+				//TODO:: (use getAllElements in selectorUtils instead of this function.
```
Comment (ibatta) [inline src/com/generic/page/HomePage.java]: Please do this change

File: src/com/generic/page/Cart.java (hunk 1 above)
Comment (walahamad) [reply src/com/generic/page/Cart.java]: Done.

File: src/com/generic/page/Cart.java (hunk 2 above)
Comment (walahamad) [reply src/com/generic/page/Cart.java]: Done.

File: src/com/generic/page/HomePage.java (hunk 3 above)
Comment (walahamad) [reply src/com/generic/page/HomePage.java]: Done.

File: src/com/generic/page/HomePage.java (hunk 4 above)
Comment (walahamad) [reply src/com/generic/page/HomePage.java]: Done.

File: src/com/generic/page/HomePage.java (hunk 5 above)
Comment (walahamad) [reply src/com/generic/page/HomePage.java]: Done.

File: src/com/generic/page/HomePage.java (hunk 6 above)
Comment (walahamad) [reply src/com/generic/page/HomePage.java]: Done.

File: src/com/generic/page/HomePage.java (hunk 7 above)
Comment (walahamad) [reply src/com/generic/page/HomePage.java]: Done.

**Pull request merged**